### ISSUE TITLE
Bug: Client server settings reset 

### ISSUE THREAD

**Issue by DRMPN** [DamianMorozov/TgDownloader#22]
I have the application completely set up.
After clicking on "Manual download" and downloading the media, the client settings are reset and you have to enter them again.

**Comment by DamianMorozov** [comment]
Can you describe the menu selection procedure in more detail?

**Comment by DamianMorozov** [comment]
My quick setup for `Download settings` looks like this:
1. `Setup source (ID/username)`
2. `Setup download folder`
3. `Manual download`

**Comment by DRMPN** [comment]
Yes, after you press `Manual download ` and wait to finish its work, you'll have to set up `client settings` again. It's not supposed to be like this, right?

**Comment by DamianMorozov** [comment]
Can you show the screenshots with comment?
